server/src/main/scala/za/co/absa/atum/server/api/http/Routes.scala

```diff
@@ -111,6 +111,26 @@ trait Routes extends Endpoints with ServerOptions {
           PartitioningController.getFlowPartitionings(flowId, limit, offset)
         }
       ),
+      createServerEndpoint[
```

Comment:
add those two endpoints to the list in the createSwaggerRoutes method so they become part of swagger docs

Reply:
Added